staging/src/k8s.io/client-go/discovery/discovery_client_test.go

```diff
@@ -484,8 +483,6 @@ func openapiSchemaDeprecatedFakeServer(status int, t *testing.T) (*httptest.Serv
 			return
 		}
 
-		mime.AddExtensionType(".pb-v1", "application/com.github.googleapis.gnostic.OpenAPIv2@68f4ded+protobuf")
```

Comment:
This is not used because we don't use `http.ServeContent`, we directly write the bytes without setting the content-type. Should we set the content types manually here and below?